### PR TITLE
Tobi/update schema

### PR DESCRIPTION
**Pull request by tobikuhlmann**

### CODE REVIEW

**Comment by tobikuhlmann** [comment]
updating the schema as it's currently running in production, required some changes

File: schema.json [hunk 1]
```diff
@@ -843,5 +813,20 @@
     "name": "updatePeriod",
     "type": "STRING",
     "mode": "NULLABLE"
+  },
+  {
+    "name": "min",
```
Comment (pedro-clabs) [inline schema.json:818]: My bad. Thanks for adding this one!

File: schema.json [hunk 2]
```diff
@@ -774,26 +764,6 @@
...
-  {
-    "name": "updatePeriod",
-    "type": "INTEGER",
-    "mode": "NULLABLE"
-  },
-  {
-    "name": "factorLastUpdated",
-    "type": "INTEGER",
-    "mode": "NULLABLE"
-  },
```
Comment (pedro-clabs) [inline schema.json:796]: Are these not being used?

Comment (tobikuhlmann) [reply schema.json:796]: They were duplicated in the schema, once as string and once as integer or float, with data in the string columns. No idea what happened there

Comment (pedro-clabs) [reply schema.json:796]: Ouch! Thanks for fixing :)

**Review by pedro-clabs — APPROVED** [review]
LGTM.

**Pull request merged**